packages/sdk-components-react/src/theme/tailwind-classes.ts

```diff
@@ -0,0 +1,203 @@
+/**
+ * Quik and dirty implementation of tailwind classes conversion to webstudio styles.
+ */
+import type { EmbedTemplateStyleDecl } from "@webstudio-is/react-sdk";
+import * as variables from "./radix-variables";
+import { theme } from "./tailwind-theme";
+import { parseCssValue } from "@webstudio-is/css-data";
+
+// https://github.com/tailwindlabs/tailwindcss/blob/master/src/css/preflight.css
+export const preflight = (): EmbedTemplateStyleDecl[] => [
+  {
+    property: "borderTopStyle",
+    value: { type: "keyword", value: "solid" },
+  },
+  {
+    property: "borderRightStyle",
+    value: { type: "keyword", value: "solid" },
+  },
+  {
+    property: "borderBottomStyle",
+    value: { type: "keyword", value: "solid" },
+  },
+  {
+    property: "borderLeftStyle",
+    value: { type: "keyword", value: "solid" },
+  },
+
+  {
+    property: "borderTopColor",
+    value: variables.border,
+  },
+  {
+    property: "borderRightColor",
+    value: variables.border,
+  },
+  {
+    property: "borderBottomColor",
+    value: variables.border,
+  },
+  {
+    property: "borderLeftColor",
+    value: variables.border,
+  },
+];
+
+export const z = (value: number): EmbedTemplateStyleDecl[] => [
+  {
+    property: "zIndex",
+    value: { type: "unit", value, unit: "number" },
+  },
+];
+
+export const overflow = (
+  value: "hidden" | "visible" | "scroll" | "auto"
+): EmbedTemplateStyleDecl[] => [
+  {
+    property: "overflow",
+    value: { type: "keyword", value },
+  },
+];
+
+export const rounded = (value: "md"): EmbedTemplateStyleDecl[] => [
+  {
+    property: "borderTopLeftRadius",
+    value: variables.radius,
+  },
+  {
+    property: "borderTopRightRadius",
+    value: variables.radius,
+  },
+  {
+    property: "borderBottomRightRadius",
+    value: variables.radius,
+  },
+  {
+    property: "borderBottomLeftRadius",
+    value: variables.radius,
+  },
+];
+
+export const border = (borderWidth?: number): EmbedTemplateStyleDecl[] => {
+  const key = `${borderWidth ?? "DEFAULT"}`;
+  const valueString = theme("borderWidth")?.[key] ?? "1px";
```

Comment:
shoudln't the theme rather have an already parsed css value so you don't need to parse it below?

Reply:
It's naked tailwind theme, can spend a lot of time parsing it fully. Here is easier as I use known props.

Reply:
hmm, so we can't use any tailwind them or can we?

Reply:
We can use any, here just supersmall theme wrapper, without any deep knowledge. Like I know that we can parse borderWidth or paddings but not sure about everything, its like Im adding utils I see in radix styles